client/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';

```

Comment:
Is this related to updating to the newest version of react?

Reply:
You're right. Yeah, we don't need this noise.

![Screen Shot 2022-10-17 at 6 58 01 PM](https://user-images.githubusercontent.com/4698973/196298935-f1785e3a-a2be-41e5-a1d1-5dd3ce054596.png)